client/my-sites/checkout/src/components/refund-policies.tsx

```diff
@@ -187,6 +193,7 @@ export function getRefundWindows( refundPolicies: RefundPolicy[] ): RefundWindow
 			case RefundPolicy.DomainNameRegistration:
 			case RefundPolicy.DomainNameRegistrationBundled:
 			case RefundPolicy.DomainNameRenewal:
+			case RefundPolicy.DomainNameTransfer:
```

Comment:
Do we need to add this here (`getRefundWindows` method)? Because transfers are actually non-refundable, so they shouldn't return a refund window I guess 🤔

I tested without this `case` and the transfer refund policy still appears in checkout.